packages/gafl-webapp-service/src/processors/__tests__/recurring-pay-helper.spec.js

```diff
@@ -53,7 +53,7 @@ describe('isRecurringPayment', () => {
     show     | agreementId  | expected
     ${true}  | ${'foo123'}  | ${true}
     ${true}  | ${undefined} | ${false}
-    ${false} | ${'foo123'}  | ${false}
+    ${false} | ${'foo123'}  | ${true}
```

Comment:
I don't think we need to be testing the value of SHOW_RECURRING_PAYMENTS at all if it no longer has any bearing on the code, so reckon you can simplify this test a bit